packages/components/src/index.ts

```diff
@@ -1498,3 +1498,4 @@ export type {
     LabelPrintingProviderProps,
     LabelPrintingContextProps,
 } from './internal/components/labels/LabelPrintingContextProvider';
+export type { SamplesEditableGridProps } from './internal/sampleModels';
```

Comment:
looks like this wasn't previously being exported from ui-components. did your related PR for ELN add a usage?

Reply:
It was: https://github.com/LabKey/labkey-ui-components/pull/1031/files#diff-dff50c64053300edab3c7500cea717c6586c423876268a20bfcb7e80f4e28aebL160